src/main/java/net/openhft/chronicle/threads/EventGroup.java

```diff
@@ -76,6 +76,7 @@ public class EventGroup
     private final Pauser replicationPauser;
     private VanillaEventLoop replication;
 
+    @SuppressWarnings("deprecation")
```

Comment:
suggest that we add a comment and a `TODO:` if we are suppressing deprecation warnings. We need to make sure that these are temporary.